js/base/Exchange.js

```diff
@@ -1359,6 +1408,10 @@ module.exports = class Exchange {
         return this.safeValue (units, decimals)
     }
 
+    addFloat (value1, value2) {
```

Comment:
there's a `this.sum` function for this

Reply:
I was not aware of `this.sum` and don't see how it gets pulled into `Exchange.js` but it works, thank you for the tip! I have removed this function from the base Exchange class.